### PR TITLE
feature/logoLoading done - change the styles of containers from compo…

### PR DESCRIPTION
**Pull request by WS-Jedp**
…nents, added a class for it.

### CODE REVIEW

File: DesignSystem/src/components/Atoms/LogoLoading.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const LogoLoading = () => (
+  <div className="logoLoading">
+      <div className="logoLoading-planet"></div>
+      <div className="logoLoading-orbit">
+        <div className="logoLoading-orbit-satelite"></div>
+      </div>
+  </div>
```
Comment (onlyabdiel) [inline DesignSystem/src/components/Atoms/LogoLoading.jsx:9]: I prefered to use the resources svg to do this but let's leave it like that

File: DesignSystem/src/containers/Atoms.jsx [hunk 2]
```diff
@@ -3,6 +3,9 @@ import SyntaxHighlighter from 'react-syntax-highlighter';
 import { dark } from 'react-syntax-highlighter/dist/esm/styles/prism';
 import components from './components';
 
+// LogoLoading
+import LogoLoading from '../Components/Atoms/LogoLoading';
```
Comment (onlyabdiel) [inline DesignSystem/src/containers/Atoms.jsx]: The import doesn't work. It can't resolve the path

File: DesignSystem/src/styles/main.scss [hunk 3]
```diff
@@ -1,10 +1,14 @@
 @import './reset';
 @import './variables';
 
+// Animations
+@import './Animations/animations.scss';
```
Comment (onlyabdiel) [inline DesignSystem/src/styles/main.scss]: The folders should be in lowercase. The files should be capital case

File: DesignSystem/src/styles/main.scss [hunk 4]
```diff
@@ -1,10 +1,14 @@
...
 @import './variables';
 
+// Animations
+@import './Animations/animations.scss';
+
 // Components
 @import './components/atoms';
 @import './components/Sidebar';
 @import './components/Buttons.scss';
+@import './Components//logoLoading.scss';
```
Comment (onlyabdiel) [inline DesignSystem/src/styles/main.scss]: This import can't resolve.

**Review by onlyabdiel — COMMENTED** [review]
I recommend you to try the project after send your pull request.

File: DesignSystem/src/components/Atoms/LogoLoading.jsx (hunk 1 above)
Comment (WS-Jedp) [reply DesignSystem/src/components/Atoms/LogoLoading.jsx:9]: Okey

File: DesignSystem/src/containers/Atoms.jsx (hunk 2 above)
Comment (WS-Jedp) [reply DesignSystem/src/containers/Atoms.jsx]: It's working here... Doesn't show any error in my Pc.
Captura

File: DesignSystem/src/styles/main.scss (hunk 3 above)
Comment (WS-Jedp) [reply DesignSystem/src/styles/main.scss]: Fixed the name of the file, but all our folders are in Capital Case, we agreed on that before when we started. I think that the problem it's that GitHub doesn't recognize which folder is Capital case or not.
Captura

File: DesignSystem/src/styles/main.scss (hunk 4 above)
Comment (WS-Jedp) [reply DesignSystem/src/styles/main.scss]: Fixed

File: DesignSystem/src/styles/main.scss (hunk 3 above)
Comment (onlyabdiel) [reply DesignSystem/src/styles/main.scss]: For that reason you should use lower case in folders.

**Review by onlyabdiel — APPROVED** [review]

**Pull request merged**